http-server/src/main/java/io/airlift/http/server/HttpServerBinder.java

```diff
@@ -57,6 +57,21 @@ public HttpServerBinder enableLegacyUriCompliance()
         return this;
     }
 
+    public void withFeatures(HttpServerFeatures serverFeatures)
```

Comment:
Can you make this API change a separate commit or PR? It's not clear how this is related to HTTP/2.